.bazelrc

```diff
@@ -61,5 +61,8 @@ build:libc++ --action_env=CXXFLAGS=-stdlib=libc++
 build:libc++ --action_env=PATH
 build:libc++ --define force_libcpp=enabled
 
+# Optimize build for binary size reduction.
+build:sizeopt --copt -Os
```

Comment:
Does it work to just set `-c opt` here? I vaguely recall this doesn't work for some reason but it might be worth a shot. Then it can be removed from elsewhere.

Reply:
Looks like it works. Version shows release in the absence of `-c opt`:
```
tallen@envoy-buildbox:~/envoy$ build_sizeopt_stripped/envoy --version

build_sizeopt_stripped/envoy  version: 1b8a474fb9f22b889417175387af67d16a1930c3/1.11.0-dev/Modified/RELEASE/BoringSSL
```